packages/scripts/src/run-environment.ts

```diff
@@ -46,17 +46,18 @@ export interface RunningFeatureOptions<F extends FeatureClass, ENV extends AnyEn
     feature: F;
 }
 
-export async function runEngineEnvironment<ENV extends AnyEnvironment>({
-    featureName,
-    configName,
-    bundlePath,
-    runtimeOptions = {},
-    config = [],
-    env,
-    basePath = process.cwd(),
-    featureDiscoveryRoot,
-}: IRunNodeEnvironmentOptions<ENV>): Promise<RuntimeEngine<ENV>> {
-    const { env: envName, envType } = env;
+function memoize<T extends (...args: string[]) => any>(fn: T): T {
+    const cache = new Map<string, ReturnType<T>>();
```

Comment:
WeakMap ?

Reply:
not for strings...

Reply:
also there is not reason here it will only run once and during tests.